README.md

```diff
@@ -222,6 +123,7 @@ A few important notes to keep in mind
 
 - [rubyowo](https://github.com/rubyowo) - CI and docs
 - [braheezy](https://github.com/braheezy) - Instructions for the GDM theme.
+- [Omar](https://github.com/somerand0mcat) - Improving the instructions
```

Comment:
IMO one docs PR isn't really enough to add yourself to the credits 😅

Reply:
oh

Reply:
i guess i'll just remove the credit then